plugins/autocertkit

```diff
@@ -1203,6 +1203,18 @@ def remove_route(session, args):
     return json_dumps("OK")
 
 
+@log_exceptions
+def set_kernel_parameter(key, value, mip=None):
+    """
+    Delete all line which contains key in config file, and then write key=value to file,
+    Will take effect after reload(sysctl --sytstem) or reboot
+    """
+    conf_file = "/etc/sysctl.d/99-customer.conf"
```

Comment:
could be `/etc/sysctl.d/ack.conf`, or just modify `/etc/sysctl.conf` directly as it the last config file to load.